lib/bookstore-scrapper/indexTextbooks.ts

```diff
@@ -0,0 +1,45 @@
+import { CampusStoreEntry } from "./helpers/getTextbookInformation";
+import { updateDatabase } from "./database/updateDatabase";
+import { getBookData } from "./helpers/getBookData";
+import { axiosRequest } from "./helpers/axiosRequest";
+import {
+  updateAlgoliaIndex,
+  getAlgoliaObject,
+} from "./algolia/updateAlgoliaIndex";
+import fs from "fs/promises";
+
+// Algolia constants - FILL BEFORE RUNNING
+export const algoliaAppId: string = "";
+export const algoliaApiKey: string = "";
+export const algoliaIndexName: string = "";
```

Comment:
I think Caleb mentioned pulling these from the environment variables. Did you take a look into this?

Reply:
I misunderstood and thought that they only needed to be set in the .env.dev file. Sorry about that, will update indexTextbooks to get the values from there.